terraform/eks/container_insights_agent.tf

```diff
@@ -0,0 +1,53 @@
+data "template_file" "cluster_role_file" {
+  template = file("./container-insights-agent/cluster_role.tpl")
+  vars = {
+    NAMESPACE = "aoc-ns-${module.common.testing_id}"
```

Comment:
better have a var as namespace and construct in one place (maybe in `shared`)?

Reply:
No. There shouldn't be any hardcode elsewhere for namespace value. Updated to use reference.